/**:
  ros__parameters:
    image_scale: 1.0
    frame_id_ground: "camera_depth_optical_frame" #default: "map"
    frame_id_depth: "camera_depth_optical_frame"  #default: "camera_depth_optical_frame"
    depth_range_max: 1.0  #[m]
    is_display_open3d: False
    marker_ns: "marker"
    TSDF:
      voxel_length: 0.05  #default: 4/512 = 0.0078125
      sdf_trunc: 0.5  #default: 0.04
